src/ipaperftest/core/main.py

```diff
@@ -120,10 +151,11 @@ def clone_ansible_freeipa(self, ctx):
                 "v0.3.8",
                 "https://github.com/freeipa/ansible-freeipa.git",
             ],
+            cwd="runner_metadata",
             stdout=sp.PIPE,
         )
-        with open("ansible.cfg", "w") as f:
-            f.write(ANSIBLE_CFG_TEMPLATE.format(cwd=os.getcwd()))
+        with open("runner_metadata/ansible.cfg", "w") as f:
+            f.write(ANSIBLE_CFG_TEMPLATE.format(cwd=os.getcwd() + "/runner_metadata"))
```

Comment:
Would os.path.join() be better than concatenating strings?

Reply:
Fixed, thanks!